consensus/enclave/trusted/Cargo.toml

```diff
@@ -6,10 +6,13 @@ edition = "2021"
 description = "The MobileCoin Consensus Service's internal enclave entry point."
 license = "GPL-3.0"
 readme = "README.md"
-rust-version = "1.74.0"
+rust-version = { workspace = true }
 
 [workspace]
 
+[workspace.package]
+rust-version = "1.74.0"
+
```

Comment:
I'm hoping this change will fix dependabot failure in the enclaves, https://github.com/mobilecoinfoundation/mobilecoin/network/updates/746370129

```
error: failed to load manifest for dependency `mc-attest-core`

Caused by:
  failed to parse manifest at `dependabot_tmp_dir/attest/core/Cargo.toml`

Caused by:
  error inheriting `rust_version` from workspace root manifest's `workspace.package.rust_version`

Caused by:
  failed to find a workspace root
```